### PR TITLE
fix the ipython type check codes

### PR DESCRIPTION
**Pull request by qinglonger**
I came from the MOOC in "Machine Learning in Weather & Climate" .  Because the code in  tiger 2 'Introduction to data handling'  don't work as we think. The figure don't show in the jupyter lab, So I check the code, and find the bug in the 'climetlab.core.ipython.py '. Bug and Fix list:
1:  The ipython_active do not get the handle of  IPython ; 2:  The code of Jupyter check  is write in error ,Bucause the ipython_dir is not match the specified one.

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/climetlab?pullRequest=54) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/climetlab?pullRequest=54) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/climetlab?pullRequest=54) it.</sub>

**Comment by floriankrb** [comment]
Thank you for finding and fixing the issue.

What about moving your additional if last in the list though. Just before ``return ("unknown", None)``. To make sure it does not break the previous ones which are more specific?

File: climetlab/core/ipython.py [hunk 1]
```diff
@@ -42,7 +53,12 @@ def guess_which_ipython():
     if ipython_active.ipython_dir == "/deepnote-config/ipython":
         return ("deepnote", None)
 
-    if ipython_active.ipython_dir == "/home/jovyan/.ipython":
+    # if ipython_active.ipython_dir == "/home/jovyan/.ipython":
+    #     return ("jupyter-lab", None)
+
+    # modify by longtsing
```
Comment (floriankrb) [inline climetlab/core/ipython.py:59]: author names are in the git commit, no need to add it in the code.

Comment (qinglonger) [reply climetlab/core/ipython.py:59]: ok,please delete it and merge into the master

**Comment by qinglonger** [comment]
> 

thanks ，I will take some hours to recheck it

> Thank you for finding and fixing the issue.
> 
> What about moving your additional if last in the list though. Just before `return ("unknown", None)`. To make sure it does not break the previous ones which are more specific?

**Pull request closed without merging**